### ISSUE TITLE
Add a study/experiment/signature via API

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDB#134]
Qiita (https://qiita.ucsd.edu/) is a popular open-source microbial study management platform that researchers use to analyze their metagenomic data. They are developing a differential abundance workflow and have expressed interest in having it post the output of a completed workflow programmatically to bugsigdb.org (this would be pre-publication).  Such a workflow could help a lot with having researchers enter their own data into bugsigdb rather than only us doing it after publication.

Is this feasible through the default API (https://bugsigdb.org/Special:ApiSandbox) or what would be involved in supporting this?

**Comment by lgeistlinger** [comment]
This actually sounds pretty cool! Can this be implemented @tosfos?

**Comment by tosfos** [comment]
I believe so. We'll check what's involved.

**Comment by tosfos** [comment]
We're going to send some instructions. It should be fairly simple.

**Comment by tosfos** [comment]
The instructions are ready. Where would be the best place for them? We can post them here, save them on a Help page in the wiki, or prepare a PDF.

**Comment by lgeistlinger** [comment]
I think a help page in the wiki will be best @lwaldron?

**Comment by lwaldron** [comment]
Yes I agree, a help page on the wiki, then we'll link to it from the ?
page.

**Comment by tosfos** [comment]
The Help page was created [here](https://bugsigdb.org/Help:Editing_using_the_API) Please let me know if you have any questions.
